pf/tests/provider_configure_test.go

```diff
@@ -20,71 +20,110 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
 	"github.com/pulumi/providertest/replay"
+	"github.com/pulumi/pulumi-terraform-bridge/pf/tests/internal/cross-tests"
 	"github.com/pulumi/pulumi-terraform-bridge/pf/tests/internal/testprovider"
 	tfpf "github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 	"github.com/stretchr/testify/require"
+	"github.com/zclconf/go-cty/cty"
 )
 
 func TestConfigure(t *testing.T) {
```

Comment:
I think the fact that the previous tests were renamed causes the diff here to display pretty badly